tox.ini

```diff
@@ -398,51 +176,21 @@ commands_pre =
   pip install -e {toxinidir}/opentelemetry-api \
               -e {toxinidir}/opentelemetry-sdk \
               -e {toxinidir}/opentelemetry-instrumentation \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-requests \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-wsgi \
-              -e {toxinidir}/instrumentation/opentelemetry-instrumentation-flask
+              -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-requests \
+              -e {toxinidir}/opentelemetry-python-contrib/instrumentation/opentelemetry-instrumentation-wsgi
```

Comment:
is flask missing here?

Reply:
I removed it because I noticed that `tests/w3c_tracecontext_validation_server.py` didn't use `opentelemetry.instrumentation.flask`. Not sure if it uses it in auto-instrumentation but I will experiment with it today to find out if it's needed.

Reply:
Turns out its not needed!